starter-code/src/functions-and-arrays.js

```diff
@@ -48,6 +120,21 @@ var wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(arr4) {
```

Comment:
With this filter (we will see it tomorrow), basically:
 We iterate over the array and check if the first position of this element in the array is equal to the current position. For duplicate elements, these two positions are different.